internal/projects/create.go

```diff
@@ -82,13 +81,6 @@ func ProvisionSelfEnrolledProject(
 		UpdatedAt:   timestamppb.New(project.UpdatedAt),
 	}
 
-	if err != nil {
```

Comment:
turns out this was a leftover and was never triggered. `nil != nil` is not achievable. Somehow this was missed and it's just dead code.